Automattic/pocket-casts-android
# [Paywall Experiment] Add Upgrade Layout Reviews part 1

## Description
- This adds the first part of the new paywall layout reviews. In the second part, I will add the list of the reviews
- Updates the existing icons displayed in cards. See: p1724423307147529-slack-C07G1MH890E
- See figma: SEQe6JBDCeegcpa9Bfg5Ww-fi-4_1455

Fixes #2709

## Testing Instructions
1. Apply this [paywall.patch](https://github.com/user-attachments/files/16612993/paywall.patch) and run in `debug`
2. Have `PAYWALL_AB_EXPERIMENT` feature flag enabled
3. Open paywall
4. ✅ Ensure the layout works as expected in Figma

## Screenshots or Screencast 

<img src="https://github.com/user-attachments/assets/500476c5-58f3-4f5a-aa54-3eea271c9f57" width="400">

## Checklist
 
#### I have tested any UI changes...
